modules/core/src/main/scala/org/tessellation/domain/cell/L0Cell.scala

```diff
@@ -85,10 +94,18 @@ object L0Cell {
 
   object Coalgebra {
 
-    def processDAGL1[F[_]: Async](data: Signed[L1Output]): CoalgebraR[F] = {
+    def processDAGL1[F[_]: Async: Logger](data: Signed[L1Output], collateral: Collateral[F]): CoalgebraR[F] = {
       def res: StackF[CoalgebraCommand] = Done(AlgebraCommand.EnqueueDAGL1Data(data).asRight[CellError])
-
-      res.pure[F]
+      def errorRes: F[StackF[CoalgebraCommand]] =
+        Logger[F]
+          .debug(s"Peers which signed block sent by ${data.proofs.head.id} have not sufficient collateral")
```

Comment:
each CellError is being logged (see: StateChannel.scala lines 105:130), so looks like this is a duplicated log

Reply:
`StateChannel` logs messages from `BlockConsensusCell`. I can't find similar code for `L0Cell`.